plugin-enclave-gradle/src/main/kotlin/com/r3/conclave/plugin/enclave/gradle/LinuxExec.kt

```diff
@@ -50,6 +51,8 @@ open class LinuxExec @Inject constructor(objects: ObjectFactory) : ConclaveTask(
                     "--tag", tag.get(),
                     "--build-arg",
                     "jep_version=$JEP_VERSION",
+                    "--build-arg",
+                    "gramine_version=$GRAMINE_VERSION",
```

Comment:
I think the solution is to simply hardcode the version here, since we don't need it anywhere else.